compass/landice/mesh.py

```diff
@@ -252,8 +319,21 @@ def get_dist_to_edge_and_GL(self, thk, topg, x, y, window_size=1.e5):
         Distance from each cell to the grounding line
     """
     logger = self.logger
+    section = self.config[section]
     tic = time.time()
 
+    high_dist = float(section.get('high_dist'))
```

Comment:
It seems like it could potentially make this function more fragile to have it use a mix of passed arguments and params pulled from a config file.  Would it be problematic to make `high_dist` and `high_dist_bed` as input arguments that get pulled from the cfg by the calling function?

Reply:
It would not necessarily be problematic, but this usage reflects `set_cell_width()`, for which would definitely be problematic (or at least annoying) to pass all the config settings from the calling function, since that would add an extra ten function parameters to the nine that are already necessary. I'd prefer to leave this as is.